pkg/core/native/native_nep17.go

```diff
@@ -128,7 +128,14 @@ func addrToStackItem(u *util.Uint160) stackitem.Item {
 }
 
 func (c *nep17TokenNative) postTransfer(ic *interop.Context, from, to *util.Uint160, amount *big.Int,
-	data stackitem.Item, callOnPayment bool) {
+	data stackitem.Item, callOnPayment bool, postCalls ...func()) {
+	defer func() {
```

Comment:
I'm not sure, but if an error is returned from OnNEP17Payment call then `defer` will still be invoked. Is it the desired behaviour? Although the transaction will be FAULTed and it's not very important.

Reply:
Fixed.